simplq/src/styles/navbar.module.scss

```diff
@@ -65,9 +65,11 @@
     }
 
     // In case of smaller screens with sidebar, display at the bottom of sidebar
-    bottom: 1rem;
+    @media (max-width: 787px) {
+      bottom: 1rem;
+    }
     @media (min-width: 787px) {
```

Comment:
Was blocking buttons on firefox, but strangely not on chrome. Have to figure out why these certain styles act differently.

![Screenshot from 2020-10-28 01-20-40](https://user-images.githubusercontent.com/12583528/97354876-1169d980-18bc-11eb-99f6-0c4adf600826.png)

Reply:
I fixed it like [this](https://github.com/SimplQ/simplQ-frontend/pull/396/files#diff-e18458cc3490415771c0eabd0f4970247727d4e31c941003955a4da4aa8a5c91R62)... can you pull latest code?

Reply:
It's happening on chrome too

Reply:
The height: min-content didn't fix it on firefox on me. I guess that fixed it on chrome though.

Reply:
nice. The bottom 1rem was causing this. Interesting find.